unified-codes/apps/web/src/components/layout/DetailLayout.tsx

```diff
@@ -21,7 +21,7 @@ export const DetailLayout: DetailLayout = ({ classes, attributeList, childList,
     <Grid item className={classes?.attributeListContainer}>
       {attributeList}
     </Grid>
-    <Grid item direction="row" className={classes?.childListContainer}>
+    <Grid item className={classes?.childListContainer}>
```

Comment:
Does this effect the UI layout? I was thinking just to add a `container` prop?

Reply:
```suggestion
    <Grid container item className={classes?.childListContainer}>
```

Reply:
`container` prop works just as well visually - happy to change it.
What about the `attributeList` and `propertyList` containers, should we convert those too for consistency?

Reply:
Yup, if it doesn't change the UI appearance then go for it!

Reply:
@wlthomson according to material-UI docs: "_If true, the component will have the flex item behavior. You should be wrapping items with a container._"

how then, can you add both `item` and `container` to a `Grid`? which one is it?

Reply:
Coool 😎  - 2e977f145

Reply:
@mark-prins sorry I think my UI hadn't refreshed and I didn't see your comment when making the commit. I decided to just use a container 😸